pynamodb/connection/table.py

```diff
@@ -131,6 +131,37 @@ def get_item(self, hash_key, range_key=None, consistent_read=False, attributes_t
             consistent_read=consistent_read,
             attributes_to_get=attributes_to_get)
 
+    def rate_limited_scan(self,
+             attributes_to_get=None,
```

Comment:
Hey. I am following the format that was present throughout the file. Making this change will make this method appear odd